services/movie.py

```diff
@@ -6,16 +7,17 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str | None = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
     if genres_ids:
         queryset = queryset.filter(genres__id__in=genres_ids)
-
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
-    return queryset
+    return queryset if title is None else (queryset.filter
+                                           (title__contains=title))
```

Comment:
It is inappropriate line breaking, in case the line is too long place it in two lines with two `return` statements for better readability.